webapp/src/share.tsx

```diff
@@ -209,7 +207,7 @@ export class ShareEditor extends auth.Component<ShareEditorProps, ShareEditorSta
         return visible
             ? <Modal
                 title={lf("Share Project")}
-                className="sharedialog wide"
+                className={`sharedialog${thumbnails ? " wide" : ""}`}
```

Comment:
Ahah, was my guesstimate of 40rem just the default when it's not `.wide`? Looks same

Reply:
Yep! I was a bit mind blown by that!

<img width="224" alt="image" src="https://github.com/microsoft/pxt/assets/49178322/e215c9d2-3469-4d1a-b792-27ec4be9b405">